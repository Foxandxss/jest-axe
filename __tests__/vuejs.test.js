const { axe, toHaveNoViolations } = require('../index')

const { mount } = require("@vue/test-utils");

expect.extend(toHaveNoViolations)

describe('Vue', () => {
  it('renders correctly', async () => {
    const Image = require('./Image.vue')
    const wrapper = mount(Image)
    const results = await axe(wrapper.element);
    
    expect(() => {
      expect(results).toHaveNoViolations()
    }).toThrowErrorMatchingSnapshot()
  })

})