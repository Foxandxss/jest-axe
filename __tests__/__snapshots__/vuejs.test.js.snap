// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vue renders correctly 1`] = `
"[2mexpect([22m[31mreceived[39m[2m).toHaveNoViolations([22m[32mexpected[39m[2m)[22m

Expected the HTML found at $('#test-image') to have no violations:

<img id=\\"test-image\\" src=\\"#\\">

Received:

[31m\\"Images must have alternate text (image-alt)\\"[39m

Try fixing it with this help: https://dequeuniversity.com/rules/axe/3.1/image-alt?application=axeAPI"
`;
